modules/cas_cache/exp_obj.c

```diff
@@ -161,11 +163,13 @@ static int _cas_del_partitions(struct cas_disk *dsk)
 	return result;
 }
 
-#ifdef GENHD_FL_NO_PART_SCAN
-static int _cas_flags = GENHD_FL_NO_PART_SCAN | GENHD_FL_EXT_DEVT;
-#else
-static int _cas_flags = GENHD_FL_EXT_DEVT;
-#endif
+//#if defined GENHD_FL_NO_PART_SCAN
+//static int _cas_flags = GENHD_FL_NO_PART_SCAN | GENHD_FL_EXT_DEVT;
+//#elif defined GENHD_FL_NO_PART
+static int _cas_flags = GENHD_FL_NO_PART;
+//#else
+//static int _cas_flags = GENHD_FL_EXT_DEV;
+//#endif
```

Comment:
doesn't this need to be parametrized in configure?

Reply:
It already is, yes :)